build.assets/Makefile

```diff
@@ -69,9 +69,17 @@ ifeq ("$(DRONE)","true")
 CI := true
 endif
 ifeq ("$(CI)","true")
+# The UID/GID of the runner user on ARC runners is 1001, not 1000
+# This var is currently only set for ARC runners via https://github.com/gravitational/cloud-terraform/pull/2473
+ifeq ("$(CI_SYSTEM)","ARC")
+UID := 1001
```

Comment:
Won't that break the image generation? We use that value when creating images. If we change the default user then the generated images will also have a different default user, so when you run that image somewhere you will probably get a permission error.

Reply:
Did some spot checks and this doesn't seem to affect any customer facing images. Right now it only appears to affect buildboxes, which now also expect (in the context of ARC runners) uid/gid of 1001.